### PR TITLE
add tutorial how to use minikube in github actions as a CI step

### PR DESCRIPTION
**Pull request by medyagh**

### CODE REVIEW

File: site/content/en/docs/tutorials/setup_minikube_in_github_actions.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Setup minikube as CI step in github actions"
+linkTitle: "minikube in github actions"
+weight: 1
+date: 2020-06-02
+description: >
+  How to use minikube in github actions for testing your app?
```
Comment (priyawadhwa) [inline site/content/en/docs/tutorials/setup_minikube_in_github_actions.md:7]: Remove the question mark at the end

File: site/content/en/docs/tutorials/setup_minikube_in_github_actions.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+for more information see github actions marketplace [setup-minikube]( https://github.com/marketplace/actions/setup-minikube).
+
+## Example: build image & deploy to minikube on each PR
+
+Requirements:
+
+- a valid Dockerfile
+- a valid `deployment.yaml` file with `imagePullPolicy: Never` see bellow for an example
```
Comment (priyawadhwa) [inline site/content/en/docs/tutorials/setup_minikube_in_github_actions.md:27]: spelling: bellow / below

File: site/content/en/docs/tutorials/setup_minikube_in_github_actions.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+          curl $(minikube service example --url)
+  ```
+
+The above example workflow yaml, will do the following steps on each coming PR:
+
+1. Checks out the the source code
+2. Installs & starts minikube
+3. Tries out the cluster just by running `kubectl` command
+4. Build the docker image using minikube's docker-env feature
+5. Apply the yaml deployment yaml file minikube
```
Comment (priyawadhwa) [inline site/content/en/docs/tutorials/setup_minikube_in_github_actions.md:71]: "yaml" is in there twice

File: site/content/en/docs/tutorials/setup_minikube_in_github_actions.md (hunk 1 above)
Comment (medyagh) [reply site/content/en/docs/tutorials/setup_minikube_in_github_actions.md]: done

File: site/content/en/docs/tutorials/setup_minikube_in_github_actions.md (hunk 3 above)
Comment (medyagh) [reply site/content/en/docs/tutorials/setup_minikube_in_github_actions.md]: done

File: site/content/en/docs/tutorials/setup_minikube_in_github_actions.md (hunk 2 above)
Comment (medyagh) [reply site/content/en/docs/tutorials/setup_minikube_in_github_actions.md]: done

**Pull request merged**